src/core_scan/Core_scan.ml

```diff
@@ -556,16 +556,18 @@ let parse_equivalences equivalences_file =
    Returns a list of match results and a separate list of scanned targets.
 *)
 let iter_targets_and_get_matches_and_exn_to_errors (config : Core_scan_config.t)
-    (handle_target : target_handler) (targets : In.target list) :
+    (handle_target : target_handler) (targets : Target_location.t list) :
     Core_profiling.file_profiling RP.match_result list * Fpath.t list =
   (* The path in match_and_path_list is None when the file was not scanned *)
   let (match_and_path_list
         : (Core_profiling.file_profiling RP.match_result * Fpath.t option) list)
       =
     targets
-    |> map_targets config.ncores (fun (target : In.target) ->
-           let file = target_path target in
-           logger#info "Analyzing %s" !!file;
+    |> map_targets config.ncores (fun (target : Target_location.t) ->
+           let file = Target_location.file target in
+           let source = Target_location.source target in
+           logger#info "Analyzing %s (file is %s)" (Source.to_string source)
```

Comment:
For these comments I would be more clear about what "file" means---maybe something like "contents written to". I want a CSE to be able to know what this means without asking us.

Reply:
Done in d2a27a2ebb2cc4faec501008b313510b558cb8bd